{"name": "test_fail_delete_task", "status": "passed", "description": "Проверка ошибки при удалении задачи с некорректными заголовками", "steps": [{"name": "Обновляем заголовки на некорректные", "status": "passed", "start": 1744822007800, "stop": 1744822007800}, {"name": "Пытаемся удалить задачу", "status": "passed", "start": 1744822007800, "stop": 1744822008027}, {"name": "Возвращаем заголовки и убеждаемся, что задача осталась", "status": "passed", "start": 1744822008027, "stop": 1744822008380}], "start": 1744822007800, "stop": 1744822008380, "uuid": "1ea83d3e-edbf-4a2d-a598-1632e178e2ae", "historyId": "fca0feda91469001c721842c2aa8b58f", "testCaseId": "fca0feda91469001c721842c2aa8b58f", "fullName": "tests.api.test_tasks.TestTasks#test_fail_delete_task", "labels": [{"name": "feature", "value": "API: Работа с задачами"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "subSuite", "value": "TestTasks"}, {"name": "host", "value": "Grw-pc"}, {"name": "thread", "value": "2872-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}